src/bench_rmp.rs

```diff
@@ -0,0 +1,41 @@
+use serde::{Deserialize, Serialize};
+use criterion::{black_box, Criterion};
+
+pub fn bench<T>(name: &'static str, c: &mut Criterion, data: &T)
+where
+    T: Serialize + for<'de> Deserialize<'de>,
+{
+    const BUFFER_LEN: usize = 50_000_000;
```

Comment:
Could you check to see if this could go a little lower? I think only cbor and serde_json use 50M buffers because they're pretty large formats. I think I also lowered the input sizes after I bumped their buffer sizes up, so I'll shrink their buffer sizes when I run the next set of benchmarks.

If it works at 10M, that would be perfect. If it doesn't fit in 10M then feel free to leave it how it is.

Reply:
I'll check. It was just copied from the cbor benchmark considering they are similar formats.

Reply:
No issues with a buffer size of 10M.